libkbfs/key_manager.go

```diff
@@ -641,8 +641,9 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 	if !isWriter {
 		if len(newReaderUsers) > 0 || addNewWriterDevice || incKeyGen {
 			// If we're a reader but we haven't completed all the work, return
-			// RekeyIncompleteError
-			return false, nil, RekeyIncompleteError{}
+			// RekeyIncompleteError.
+			rekeyDone := len(newReaderUsers) > 0
```

Comment:
`newReaderUsers` means that a different reader has devices that need to be keyed for this directory, and our permission model doesn't allow a reader to key for other readers.